packages/native/src/components/result/ReactiveList.js

```diff
@@ -130,7 +130,10 @@ class ReactiveList extends Component {
 		if (this.props.stream !== nextProps.stream) {
 			this.props.setStreaming(nextProps.componentId, nextProps.stream);
 		}
-
+		// called when results are updated
+		if (this.props.onResultStats && nextProps.hits !== this.props.hits) {
+			this.props.onResultStats(nextProps.total, nextProps.time);
```

Comment:
We have changed the spec for this. Refer: https://github.com/appbaseio/reactivesearch/pull/603

Reply:
Please also send a PR to update relevant docs in `v3` reactive-manual 🐱